backend/routes/run.js

```diff
@@ -0,0 +1,194 @@
+import { PrismaClient } from "@prisma/client";
```

Comment:
nit: Can you maybe rename this class or add a comment at the top explaining what this is and what it does?